virttest/utils_time.py

```diff
@@ -161,10 +163,10 @@ def execute(cmd, timeout=360, session=None):
     :return: Command output string
     """
     if session:
-        ret = session.cmd_output(cmd, timeout=timeout)
+        ret = session.cmd_output_safe(cmd, timeout=timeout)
     else:
         ret = process.getoutput(cmd)
-    target = session and "guest" or "host"
+    target = 'guest' if session else 'host'
```

Comment:
Their logic are same. Unnecessary change.

Reply:
Thanks, yes, the logic is same, I update it as `pylint` suggest me to, so update it as well.